### DISCUSSION TITLE
Bugs in Zelda Ocarina of Time

### DISCUSSION THREAD

**Discussion by ghost** [mupen64plus-ae/mupen64plus-ae#985] [Q&A] [answered]
Hi, I have been experiencing a texture bug in Zelda Ocarina of Time. Whenever I open the game's inventory, it takes about 5 seconds to enter and it gets this bug there in the image of the link, and in version 0.295 the menu opened normally and there was no such bug, but after the updates, it's kind of buggy.

![Screenshot_20211208-122214](https://user-images.githubusercontent.com/95763795/145236318-7d807178-77b9-452d-a05e-1bad2e3119da.jpg)

**Comment by fzurita** [comment, marked as answer]
This is a bug with glide64mk2, even before the update, it has not been fully reliable. To fix, select the game, go to settings, and change the emulation profile to GLideN64-Medium.

**Reply by ghost** [reply]
Thanks Bro!
